DGP_CVXPY_Code/CS_amp_optimize_BW.py

```diff
@@ -0,0 +1,61 @@
+import cvxpy as cp
+import numpy as np
+
+# Constants
+VTO = 0.5 # Threshold voltage

```

Comment:
As previously requested, please clean up your comments. Why specify the units for some valuables and not for others? Why randomly specify them in parentheses versus not. Please try to become more professional in your coding.